build.gradle

```diff
@@ -524,6 +525,7 @@ project('spring-integration-core') {
     apply plugin: 'org.jetbrains.dokka'
 
     dependencies {
+        api "com.google.protobuf:protobuf-java:$protobufVersion"
```

Comment:
This has to be an `optionalApi` and therefore move it to respective section of the dependencies.
The framework by itself doesn't use this library directly, like we do, for example, with `reactor-core`, so this `protobuf` has to be realigned same way as we do for `avro`.